app/main.py

```diff
@@ -1,13 +1,39 @@
+# Импорты должны находиться на самом верху файла (исправляем E402)
+from typing import List, Dict
+
+# Добавляем две пустые строки перед началом функции (исправляем E302)
+
+
```

Comment:
Two empty lines are correctly added before the start of the function `format_linter_error`, which resolves the E302 issue.